### PR TITLE
[ADF-1867] Typography Documentation

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Typography Documentation

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
@therealandeeee can you review it ? thanks

File: docs/theming.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Available pre-built themes:
 * `adf-pink-bluegrey.css`
 * `adf-purple-green.css`
 
-If you're using Angular CLI you can include one of the prebuilt theme in yours `styles.css` file:
+If you're using Angular CLI you can include one of the prebuilt theme in yours `styles.scss` file:
```
Comment (therealandeeee) [inline docs/theming.md]: Should be "...include one of the prebuilt theme**s** in **your** `style.css` file"

File: docs/typography.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# ADF Typography
+
+## What is typography?
+
+Typography configuration let you change the style of the text in your ADF app
```
Comment (therealandeeee) [inline docs/typography.md]: "...let**s** you change..."

File: docs/typography.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+```html
+  <link href="https://fonts.googleapis.com/css?family=Muli" rel="stylesheet">
+```
+
+After you need to change your `/src/custom-style.scss` to include the new font:
+
+```scss
+/*
+ *  Include only packages what you are using (and core by default)
```
Comment (therealandeeee) [inline docs/typography.md]: "...packages **that** you are using..."

File: docs/typography.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+@include alfresco-activity-tasklist-theme($theme);
+@include alfresco-datatable-theme($theme);
+@include alfresco-documentlist-theme($theme);
+@include alfresco-login-theme($theme);
+@include alfresco-upload-theme($theme);
+@include alfresco-userinfo-theme($theme);
+
+```
+
+for more details about typografy refer to [Material 2 documentation](https://github.com/angular/material2/blob/master/guides/typography.md)
```
Comment (therealandeeee) [inline docs/typography.md]: Should be "ph" in "typography".

**Comment by therealandeeee** [comment]
I've added a few comments to the files. Also, you should ideally add typography.md to the summary.json file in the doc folder so that Compodoc and the index script can pick it up. If it is easier for you then I'm happy for you to just merge the files as they are and I'll fix them up next time I make a PR (prob later today).

**Comment by eromano** [comment]
done

**Pull request merged**